deploy/charts/operator/values-openshift.yaml

```diff
@@ -12,12 +12,12 @@ operator:
   # -- List of image pull secrets to use
   imagePullSecrets: []
   # -- Container image for the operator
-  image: ghcr.io/stacklok/toolhive/operator:v0.2.0
+  image: ghcr.io/stacklok/toolhive/operator:v0.2.8
```

Comment:
I /think/ this was fixed in a separate PR and rebasing atop origin/main should just pull in those changes so you don't have to include them in yours.